incorporating-accessibility-into-your-dev-process/focus-tests/solution/src/index.tsx

```diff
@@ -2,11 +2,12 @@ import React from "react";
 import { createRoot } from "react-dom/client";
 
 import App from "./App";
+import "./styles.css";
 
 const mainElement = document.querySelector("main");
 if (mainElement) {
-    const root = createRoot(mainElement);
-    root.render(<App />);
+  const root = createRoot(mainElement);
+  root.render(<App />);
 } else {
-    console.error("No main element");
-}
\ No newline at end of file
+  console.error("No main element");
```

Comment:
this log needed?

Reply:
Yes, there should be a log when there’s an error.